vmdb/app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -46,7 +46,7 @@ def textual_type
   end
 
   def textual_port
-    ["openstack", "openstack_infra", "rhevm"].include?(@ems.emstype) ? {:label => "API Port", :value => @ems.port} : nil
+    @ems.class.supports_port? ? {:label => "API Port", :value => @ems.port} : nil
```

Comment:
Should an `ems` have an instance method `supports_port?` that delegates to the class method?  Because, conceptually, there is nothing wrong with asking the EMS itself if it `supports_port?`